### DISCUSSION TITLE
Build multiple deb packages for Debian versus Ubuntu?

### DISCUSSION THREAD

**Discussion by thediveo** [goreleaser/goreleaser#4242] [Q&A] [answered]
Can `goreleaser` build multiple `.deb` packages, one for Debian and another one for Ubuntu? Or for different versions of Ubuntu?

The reason is that my Go binary needs to be integrated into the particular distribution; in particular, a desktop (thanks to XDG, the specific desktop doesn't matter). Unfortunately, Debian und Ubuntu vary widely when it comes to the package dependencies and also location of certain files. It seems that distro packagers are extremely keen to come up with their own favorite location that no-one else uses.

How do I create multiple `.deb` packages with `goreleaser`?

**Comment by caarlos0** [comment, marked as answer]
`nfpms` is an array, you can add multiple configurations there, as long as the final filenames are different, it should be fine.

https://goreleaser.com/customization/nfpm/

**Reply by thediveo** [reply]
Luckily, it turns out that Debian and Ubuntu at the moment does not differ in the deb configuration in this particular case. Phew. But good to know how to deal with such situations in the future. Thank you very much!
